compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -166,25 +166,32 @@ pub fn time_trace_profiler_finish(file_name: &str) {
 // Though note that Rust can also be build with an external precompiled version of LLVM
 // which might lead to failures if the oldest tested / supported LLVM version
 // doesn't yet support the relevant intrinsics
-pub fn to_llvm_feature<'a>(sess: &Session, s: &'a str) -> &'a str {
+pub fn to_llvm_feature(sess: &Session, s: &str) -> Vec<String> {
```

Comment:
Returning a `&'static [&'static str]` would simplify the code in this function below significantly. Like you can then write `&["pclmul"]` instead, which is significantly cleaner (and probably more efficient)

Reply:
How do I transform the input `&'a str` into a `&'static str`? I didn't see an obvious way to do the "no change" case.

Reply:
Ah, I totally forgot/missed that. You can still make `pub fn to_llvm_feature(sess: &Session, s: &str) -> Vec<&str> {` work though, I believe.

Reply:
Done.